atc/wrappa/compression_wrappa.go

```diff
@@ -0,0 +1,40 @@
+package wrappa
+
+import (
+	"code.cloudfoundry.org/lager"
+	"github.com/NYTimes/gziphandler"
+	"github.com/concourse/concourse/atc"
+	"github.com/tedsuo/rata"
+)
+
+type CompressionWrappa struct {
+	lager.Logger
+}
+
+func NewCompressionWrappa(logger lager.Logger) Wrappa {
+	return CompressionWrappa{
+		logger,
+	}
+}
+
+func (wrappa CompressionWrappa) Wrap(handlers rata.Handlers) rata.Handlers {
+	wrapped := rata.Handlers{}
+
+	for name, handler := range handlers {
+
+		// always gzip for events
+		if name == atc.BuildEvents {
+			gzipEnforcedHandler, err := gziphandler.GzipHandlerWithOpts(gziphandler.MinSize(0))
+			if err != nil {
+				wrappa.Logger.Debug("Failed to create gzip handler with options", lager.Data{"error": err})
```

Comment:
```suggestion
				wrappa.Logger.Error("failed-to-create-gzip-handler", err)
```

or just eliminate the logger and ignore the error since it should be impossible (it only errors on invalid options)